compiler/x/codegen/OMRTreeEvaluator.hpp

```diff
@@ -286,6 +286,15 @@ class OMR_EXTENSIBLE TreeEvaluator: public OMR::TreeEvaluator
    // routines for floating point values that can fit in one GPR
    static TR::Register *floatingPointStoreEvaluator(TR::Node *node, TR::CodeGenerator *cg);
 
+   // SIMD evaluators
+   static TR::Register *BinaryArithmeticEvaluator(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
The code around this makes it look more generic than just SIMD as does the name - is it really intended for SIMD? If so it should have SIMD in its name no?